packages/web3-modal/src/components/modals/TransactionsModal/index.tsx

```diff
@@ -124,12 +125,17 @@ export const TransactionCard = memo(function TransactionComponent({
   blockExplorerUris?: PstlWeb3ModalProps['blockExplorerUris']
   connectorMetadata?: WalletMetaData
 }) {
-  const { Icon, tooltip, reasonTooltip, ...statusStyleProps } = statusToPillProps(transaction)
+  const theme = useTheme()
+  const { Icon, tooltip, reasonTooltip, ...statusStyleProps } = statusToPillProps(
+    transaction,
+    theme?.modals?.transactions?.card?.statusPill
+  )
   const explorerUri = blockExplorerUris?.default.url
   const formattedHash = transaction.transactionHash ? truncateHash(transaction.transactionHash, { type: 'short' }) : '-'
-
   return (
-    <TransactionWrapper background={statusToCardBgColor(transaction.status)}>
+    <TransactionWrapper
+      background={statusToCardBgColor(transaction.status, theme.modals?.transactions?.card?.background)}
```

Comment:
i would change these fns to accept the same signature for both statusToX functions i.e 
```ts
statusToCardBgColor(transaction, theme)

// somewhere else
statusToPillProps(transaction, theme)
```

Reply:
and then inside these fns adjust for the new sig

Reply:
fixed: [e1007b0](https://github.com/PAST3LLE/monorepo/pull/77/commits/e1007b0acebd4eb5a37bcca3c854f026c75de83e)